src/registrar/templates/admin/import_export/change_list_import_item.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+{% load admin_urls %}
+
+{% if has_import_permission %}
```

Comment:
**Non blocking Question to confirm understanding** I don't see us overriding this function in the Resource classes, so this will just return for all authenticated admin users right?

Reply:
yes